src/core/bittorrent/torrenthandle.cpp

```diff
@@ -1055,12 +1055,12 @@ int TorrentHandle::downloadPayloadRate() const
     return m_nativeStatus.download_payload_rate;
 }
 
-int TorrentHandle::totalPayloadUpload() const
+qlonglong TorrentHandle::totalPayloadUpload() const
```

Comment:
Not really.  The typedef size_type works up to 1.0.6, but is removed in libtorrent 1.0.7, and they reference boost::int64_t.

They removed it because a conflict with std::string::size_type.  I guess it caused confusion or something.

Anyway boost::int64_t is a signed int64 and it's the same as a qint64 or a qlonglong. 

Remember we would have to change functions that are already there like torrentUpload() and torrentDownload() that are already defined as qlonglong, they just missed the Payload ones...